website/tests/test_main.py

```
@@ -318,7 +338,7 @@ def setUp(self):
 
     def test_remove_user_from_issue(self):
         # Only the issue poster can delete own issue
-        self.client.login(username="testuser", password="password")
+        self.client.login(email="test@example.com", password="password")
```

Ruff (0.14.7)

Possible hardcoded password assigned to argument: "password"

(S106)